src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -25,6 +25,13 @@ class ReferralService(
   val interventionRepository: InterventionRepository,
   val eventPublisher: ReferralEventPublisher
 ) {
+  fun assignSentReferral(referral: Referral, assignedBy: AuthUser, assignedTo: AuthUser): Referral {
```

Comment:
Remove referral as a supplied parameter and retrieve as part of method and through exception if entity not found